lib/taste_tester/tunnel.rb

```diff
@@ -44,15 +44,13 @@ def run
     end
 
     def cmd
-      if TasteTester::Config.user != 'root'
-        pid = '$$'
-      else
-        pid = '\\$\\$'
```

Comment:
This *should* be fine. taste-tester is always run as the user, but our tt-me script requires sudo to edit the config. tt-me doesn't wrap TT itself, it just does client.rb tweaking and symlinking.